### ISSUE TITLE
Email just sent out lacks hero alt text

### ISSUE THREAD

**Issue by jkjustjoshing** [StateOfJS/StateOfJS-Vulcan#153]
I just got an email with the subject "Lots of survey news! (State of JS, CSS, and GraphQL)" with this content

without loading the image | with loading the image
---- | ------
<img width="822" alt="image" src="https://user-images.githubusercontent.com/813192/195350264-e21bac85-c78f-44a5-aa21-6a9cb6f50d5b.png"> | <img width="820" alt="image" src="https://user-images.githubusercontent.com/813192/195350391-6ae12dd5-e28d-4dc4-ae4c-0efcad78ed27.png">

The image's alt text should have been filled in to reflect the content of that image.

**Comment by SachaG** [comment]
Thanks!
